test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreLoggingTests.cs

```diff
@@ -139,6 +140,101 @@ public async Task RestoreLogging_VerifyNU1608MessageAsync()
                 log.Message.Should().Contain("Detected package version outside of dependency constraint: x 1.0.0 requires z (= 1.0.0) but version z 2.0.0 was resolved.");
             }
         }
+        [Fact]
+        public async Task RestoreLogging_VerifyNU5036ErrorMessageAsync()
```

Comment:
I think the test name can be improved. 

The scenario is missing nuspec in a source, the test name should communicate that.